core/src/main/java/tc/oc/pgm/api/match/Match.java

```diff
@@ -407,6 +407,10 @@ default boolean finish() {
    */
   boolean calculateVictory();
 
+  boolean isFriendlyFire();
+
+  void setFriendlyFire(boolean enabled);
+
```

Comment:
imo this does not belong in match. MapInfo has a flag for it, if you want to modify it at runtime it should probably be part of whatever module handles it, not match itself. (maybe either scoreboard module since it handles teams, or some other damage-related module)